docs: update environment-variables.md (#662)
anitab-org/mentorship-backend: docs/environment-variables.md

- Content of `.env` file:
```

FLASK_ENVIRONMENT_CONFIG=dev
SECRET_KEY='some_random_key'
(...)
```

# Database Environment Variables

To run the backend in other than `local` mode i.e `prod` or `dev` you need to export database environment variables.
These are the required database environment variables.
```
export DB_TYPE=<database_type>
export DB_USERNAME=<database_username>
export DB_PASSWORD=<database_password>
export DB_ENDPOINT=<database_endpoint>
export DB_NAME=<database_name>
```
 ## Database environment variables description

| Environment Variable | Description                                                                 | Example           |
|----------------------|-----------------------------------------------------------------------------|-------------------|
| DB_TYPE | Type of database you want to use | postgres, mysql |
| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123 | 
| DB_PASSWORD | Database password of the `DB_USERNAME` user | mypwd123 |
| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db, remote-database-url:5432 |

> | DB_NAME | Name of the database to which you want to connect| Users.db, Tasks.db |

Reviewer: ```suggestion
| DB_NAME | Name of the database to which you want to connect | mentorship |
```

Reviewer: @divyanshugit, I gave you an example here which can be less misleading :) Please ping after you fix this little change and then I can approve

export FLASK_ENVIRONMENT_CONFIG=dev
```

> Windows user should use **set** instead of **export** for setting these environment variables. <br/>

Reviewer: No need to remove this, this line in intentional to help Windows users. Can you undo this change, please?

- Content of `.env` file:
```
FLASK_ENVIRONMENT_CONFIG=dev
SECRET_KEY='some_random_key'
(...)
```

> # Database Enviorment Variables

Reviewer: ```suggestion
# Database Environment Variables
```

To run the backend in other than `local` mode i.e `prod` or `dev` you need to export database environment variables.
These are the required database environment variables.
```
export DB_TYPE=<database_type>
export DB_USERNAME=<database_username>
export DB_PASSWORD=<database_password>
export DB_ENDPOINT=<database_endpoint>

> export DB_NAME=<database_name> |

Reviewer: ```suggestion
export DB_NAME=<database_name>
```

```
 ## Database environment variables description

| Environment Variable | Description                                                                 | Example           |
|----------------------|-----------------------------------------------------------------------------|-------------------|
| DB_TYPE | Type of database you want to use | postgres, mysql |

> | DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe | 

Reviewer: ```suggestion
| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123 | 
```

Just one example here is ok :) so that it does not confuse contributors into thinking they have to put 2 values

| DB_PASSWORD | Database password of the `DB_USERNAME` user | mypwd123 |

> | DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db |

Reviewer: ```suggestion
| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db, remote-database-url:5432 |
```

This is to give an idea that they can use a remote URL

> | DB_NAME | Name of the database to which you want to connect| Users.db, Tasks.db |

Reviewer: these variable examples are not correct since it is not about the table name, but the database name. So for example, if you use a service like ElephantSQL for example. So as examples maybe you could have `database1`? It's hard to come up with an easy to understand example 🤔
